autogen.pl

```diff
@@ -979,6 +979,26 @@ sub patch_autotools_output {
 
         # We have to change the search pattern and substitution on each
         # iteration to take into account the tag changing
+        my $search_string = '# Lahey Fortran 8.1.\n.*\n\s+' .
```

Comment:
thanks, i will double check that tomorrow.
(new) line 1026 has a different purpose, but my ```$search_string``` might match more expressions than needed